### PR TITLE
13 feature admin interface page stub

### LINKED ISSUE #13: Feature - Admin Interface - Page stub
Make html stub page for the admin interface, placing a `div` at the location of future components (e.g. the graph, header, footer, sidebar, or anything else relevant)

Most minimal version of this as a base:

Image

### PR DESCRIPTION
**Pull request by LKBEAST**
- Page stub created admin interface.
- Added gitignore

closes #13

### CODE REVIEW

File: templates/admin_interface.html [hunk 1]
```diff
@@ -1,31 +1,83 @@
...
-  <title>Hello World Setter</title>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Admin Interface</title>
+
+    <script>
+        function deleteRow(row,dd) {
+          var i = row.parentNode.parentNode.rowIndex;
+          console.log(dd)
```
Comment (met4000) [inline templates/admin_interface.html:12]: missing semicolon

File: templates/admin_interface.html [hunk 2]
```diff
@@ -1,31 +1,83 @@
...
+          document.getElementById(dd).deleteRow(i);
+        }
+        
+        function insRow(dd) {
+          console.log('hi');
```
Comment (met4000) [inline templates/admin_interface.html:17]: should this still be here?

File: templates/admin_interface.html [hunk 3]
```diff
@@ -1,31 +1,83 @@
...
-});
-Qualtrics.SurveyEngine.addOnReady(function() {
-	let input = document.querySelector(\`#QID2 > div.Inner.BorderColor.ML > div > fieldset > div > div > textarea\`);
-	input.value = "${prompt("answer to input:")}";
-});`;
-    });
-  </script>
+<body>
+    <center><h1>Admin Home Page</h1></center>
+    <ul style="    list-style-type: none;
```
Comment (met4000) [inline templates/admin_interface.html:36]: should css styles be done through classes and in a separate file?

**Review by met4000 — CHANGES_REQUESTED** [review]

File: templates/admin_interface.html (hunk 3 above)
Comment (LKBEAST) [reply templates/admin_interface.html]: Fixed the changes

**Review by LKBEAST — COMMENTED** [review]
Basically I fixed all the issues

I moved the styling into the css

I moved the javascript into the js file

I added my graph and my header to Wayne's design so we can also get rid of those tickets.

**Review by met4000 — DISMISSED** [review]

**Review by met4000 — APPROVED** [review]

**Pull request merged**